inspirehep/modules/records/mappings/records/authors.json

```diff
@@ -23,6 +23,20 @@
                     "analyzer": "lowercase_analyzer",
                     "type": "string"
                 },
+                "advisors": {
```

Comment:
So, as suggested by @jacquerie at standup, it is true that finally you can search in ES just the recid. So, since this will always contain an integer, ES will have correctly mapped by default, so there won't be a need for touching all these mappings in your PR.